README.md

```diff
@@ -29,10 +34,42 @@ optional arguments:
   -a AUTO, --auto AUTO  Automatically perform -l and then -b on a root URL
   -f, --force           Force the URLs to use the supplied socket (in case the
                         server returns localhost)
+
+$ pipenv run start -h
 ```
+
 ## Usage
+Docker mode:
+```bash
```

Comment:
It looks like setting the markdown code language as "bash" causes certain keywords (help, exit, for, then, etc) to be highlighted relative to the rest of the content. You may be able to see what I mean below in the diff, Github shows the keywords as blue or red on my screen. I'd recommend removing the language entry. "console" may also be ok but the other markdown code entries should be changed to be consistent. Your call :D

Reply:
No problem at all.  I'll update it to be just generic highlighting without the language specification.